tests/spread/general/extension-flask/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: flask extension test
+
+execute: |
+  export ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS=true
+  
+  run_rockcraft pack
+
+  test -f flask-extension_0.1_amd64.rock
+  test ! -d work
+
+  # Ensure docker does not have this container image
+  docker rmi --force flask-extension
+  # Install container
+  sudo /snap/rockcraft/current/bin/skopeo --insecure-policy copy oci-archive:flask-extension_0.1_amd64.rock docker-daemon:flask-extension:latest
+  # Ensure container exists
+  docker images flask-extension | MATCH "flask-extension"
+  
+  # ensure container doesn't exist
+  docker rm -f flask-extension-container
+  
+  # test the flask project is ready to run inside the container
```

Comment:
Thanks for this test! I think we need two more checks here, to ensure that everything is hooked up ok:
1) Run the container without overriding the entrypoint, to make sure the pebble service works
2) Run the container with a port bind and then check that the webapp is up (for example, curl the address maybe)

Reply:
example: https://github.com/canonical/rockcraft/blob/main/tests/spread/large/repo-overlay/task.yaml#L16

Reply:
Gotcha, updated, thanks!